pandas/core/frame.py

```diff
@@ -7416,46 +7422,58 @@ def quantile(self, q=0.5, axis=0, numeric_only=True,
 
         Returns
         -------
-        quantiles : Series or DataFrame
-
-            - If ``q`` is an array, a DataFrame will be returned where the
-              index is ``q``, the columns are the columns of self, and the
-              values are the quantiles.
-            - If ``q`` is a float, a Series will be returned where the
-              index is the columns of self and the values are the quantiles.
-
-        Examples
-        --------
-
-        >>> df = pd.DataFrame(np.array([[1, 1], [2, 10], [3, 100], [4, 100]]),
-                              columns=['a', 'b'])
-        >>> df.quantile(.1)
-        a    1.3
-        b    3.7
-        dtype: float64
-        >>> df.quantile([.1, .5])
-               a     b
-        0.1  1.3   3.7
-        0.5  2.5  55.0
-
-        Specifying `numeric_only=False` will also compute the quantile of
-        datetime and timedelta data.
-
-        >>> df = pd.DataFrame({'A': [1, 2],
-                               'B': [pd.Timestamp('2010'),
-                                     pd.Timestamp('2011')],
-                               'C': [pd.Timedelta('1 days'),
-                                     pd.Timedelta('2 days')]})
-        >>> df.quantile(0.5, numeric_only=False)
-        A                    1.5
-        B    2010-07-02 12:00:00
-        C        1 days 12:00:00
-        Name: 0.5, dtype: object
+        Series
 
         See Also
         --------
         pandas.core.window.Rolling.quantile
+            Returns the rolling quantile for the DataFrame.
```

Comment:
I think the `See Also` section has the format `func : desc` in the same line (continuing in the next). Can you check the docs and the other docstrings and adapt.